cypress/integration/new_task_creation_admin.js

```diff
@@ -0,0 +1,41 @@
+import Tasks from '../objects/tasks';
```

Comment:
I suggest naming files in camel case, like `newTaskCreationAdmin.js` or simply name this file  `newTaskCreation.js`